tensorflow_datasets/scripts/cli/list.py

```diff
@@ -0,0 +1,23 @@
+
+"""`tfds list` command."""
+
+import argparse
+
+from tensorflow_datasets.core import load
+
+
+def register_subparser(parsers: argparse._SubParsersAction) -> None:  # pylint: disable=protected-access
+  """Add subparser for `list` command."""
+  new_parser = parsers.add_parser('list', help='Lists all datasets')
+  new_parser.set_defaults(subparser_fn=_list_datasets)
+
+
+def _list_datasets(args: argparse.Namespace) -> None:
+  """Creates the dataset directory. Executed by `tfds new <name>`."""
+  list_datasets()
+
+
+def list_datasets():
+  datasets = load.list_builders(with_community_datasets=True)
+  for dataset in datasets:
```

Comment:
I think we should simply print the whole list as it as, just like `tfds.list_builders()` since, printing each dataset in newline will generate a pretty huge list, which might not be ideal for the user to browse.

Reply:
As mentioned in the original Issue it should be easily greppable, and following the way this is usually done, each dataset in a new line makes sense.